src/components/forms/textarea.jsx

```diff
@@ -17,7 +17,7 @@ var TextArea = React.createClass({
         return (
             <FRCTextarea {... this.props}
                          className={classes}
-                         elementWrapperClassName="textarea-group" />
+                         rowClassName="textarea-row" />
```

Comment:
not the way it was originally set up – `.textarea` is applied to style the `textarea` element itself, rather than wrappers, since it need to override defaults. We could switch it so that custom styling applies to the wrapping row, if we want to. I'd be fine with that.